src/generator/app.js

```diff
@@ -1,18 +1,32 @@
 'use strict';
 
 var express = require('express');
+var multer = require('multer');
 var app = express();
 
+var upload = multer({ dest: 'uploads/' });
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'var' statement.](https://www.codacy.com/app/triplez/open-event-webapp/file/2729341163/issues/source?bid=3233857&fileBranchId=3349103#l7)